bin/main.rb

```diff
@@ -1,3 +1,65 @@
 # frozen_string_literal: true
 
-'Hello World!'
+# !/usr/bin/env ruby
```

Comment:
Please, move it at the beginning of the file.
It have to be the first instruction.

Reply:
# !/usr/bin/env ruby  - at the first line  generated stickler issues

Reply:
Seriously ? Write me on Slack to see that please.

Reply:
nvm, it was another branch